src/server/models/listItem/listItem.ts

```diff
@@ -184,8 +150,11 @@ export async function togglerListItemIsPublished({
         where: { id },
         data: {
           isApproved: true,
-          isPublished,
+          isPublished: isPublished,
```

Comment:
Why did this not work?
```suggestion
          isPublished,
```

Reply:
accepted, thanks! I think my original intention was to refactor this so it "matched" the function name. i.e. "togglerListItemIsPublished" I'm expecting it to toggle, i.e invert the current state...so it would've been `isPublished: !isPublished,` but then a lot of functions were using this so I didn't want to break/change too many things